app/javascript/src/components/editor/EditorItemCreate.svelte

```diff
@@ -32,13 +33,13 @@
 
 <svelte:window on:click={outsideClick} on:keydown={event => { if (event.key === "Escape") active = false }} />
 
-<div class="dropdown w-100" class:dropup>
-  <button class="empty-button" on:click|stopPropagation={toggle} bind:this={element}>
-    <slot />
```

Comment:
This was using a slot so it could in the future be re-used in other places, for example create an item directly in a folder.